includes/openid-connect-generic-settings-page.php

```diff
@@ -347,24 +350,28 @@ private function get_settings_fields() {
 				'title'       => __( 'Link Existing Users', 'daggerhart-openid-connect-generic' ),
 				'description' => __( 'If a WordPress account already exists with the same identity as a newly-authenticated user over OpenID Connect, login as that user instead of generating an error.', 'daggerhart-openid-connect-generic' ),
 				'type'        => 'checkbox',
+				'disabled'    => defined( 'OIDC_LINK_EXISTING_USERS' ),
 				'section'     => 'user_settings',
 			),
 			'create_if_does_not_exist'   => array(
 				'title'       => __( 'Create user if does not exist', 'daggerhart-openid-connect-generic' ),
 				'description' => __( 'If the user identity is not link to an existing Wordpress user, it is created. If this setting is not enabled and if the user authenticates with an account which is not link to an existing Wordpress user then the authentication failed', 'daggerhart-openid-connect-generic' ),
 				'type'        => 'checkbox',
+				'disabled'    => defined( 'OIDC_CREATE_IF_NOT_EXISTS' ),
```

Comment:
@ChloeTigre you didn't document in your PR, and didn't add this to the PHPStan bootstap, adding this additional constant. Please remove this or add the missing documentation and additional support.